source/weakref/index.rst

```diff
@@ -85,7 +85,8 @@ mais le ramasse-miettes python va utiliser un détecteur de références circula
    :language: python
 
 Maintenant si nous tentons de faire des références cycliques sur notre objet. Que A a une référence sur B et que B a une référence sur A.
-Au moment où l'on veut détruire nos objets, les destructeurs de nos objets a et b ne sont jamais appelé et vont exister tant que l'interpréteur n'est pas quitté.
+Au moment où l'on veut détruire nos objets, les destructeurs de nos objets A et B ne sont jamais appelé et vont exister tant que l'interpréteur n'est pas quitté (dans le cas où
+le ramasse-miettes à été desactiver avec gc. :py:func:`~gc.disable`, pour éviter la collect automaitique de cette exemple).
```

Comment:
L'exemple des références circulaires n'est pas incroyable. Je laisserais tout simplement que Python sait le faire et qu'il n'est pas nécessaire de se prendre la tête avec `weakref` pour résoudre ce problème-là (car ce n'en est pas un).

_less is more_